NPM client is not working out of the box

If you do

```
mkdir tmp
npm init
npm install @attic/noms
node
```

and then:

```
require('@attic/noms')
```

we get `Cannot find module 'babel-runtime/helpers/classCallCheck'`

If you manually install babel-runtime it works which is not good.

Let's add `babel-runtime` to the dependencies.

--- a/js/package.json
+++ b/js/package.json
@@ -1,10 +1,13 @@
 {
   "name": "@attic/noms",
-  "version": "12.0.0",
+  "version": "12.1.0",
+  "description": "Noms JS SDK",
+  "repository": "https://github.com/attic-labs/noms",
   "main": "dist/commonjs/noms.js",
   "jsnext:main": "dist/es6/noms.js",
   "dependencies": {
     "babel-regenerator-runtime": "6.5.0",
+    "babel-runtime": "5.8.38",
     "rusha": "0.8.3",
     "text-encoding-utf-8": "1.0.1"
   },

